> polars/lazyframe/frame.py:9277: error: Argument "context" to "LazyFrameRemote" has incompatible type "ClientContext | None"; expected "ComputeContext | None"  [arg-type]                                                                                                                                                         

This is due to the fact that `LazyFrameRemote` in polars-cloud code also has the incorrect type. This will resolve itself once polars-cloud is released.

I will add a mypy ignore there for now.

--- a/py-polars/src/polars/lazyframe/frame.py
+++ b/py-polars/src/polars/lazyframe/frame.py
@@ -9274,7 +9274,7 @@ naive plan: (run LazyFrame.explain(optimized=True) to see the optimized plan)
         """
         return pc.LazyFrameRemote(
             lf=self,
-            context=context,
+            context=context,  # type: ignore[arg-type]
             plan_type=plan_type,
             n_retries=n_retries,
             engine=engine,

